unimod_mapper/unimod_mapper.py

```diff
@@ -688,6 +719,165 @@ def _reparseXML(self, xml_file_list=[]):
         self._data_list = self._parseXML(xml_file_list=xml_file_list)
         self._mapper = self._initialize_mapper()
 
+    def map_mods(self, mod_list):
+        """
+        Maps modifications defined in params["modification"] using unimod. Using the
+        dict format for the mods, the dict can be adjusted depending on the purpose of
+        the mapping. Moreover, it can only have the minimal amound of items, or e.g.
+        engine-specific ones. At the end the mapped vallues will be updated to the
+        original dict.
```

Comment:
Some typos/adjustments:
- "using unimod" --> "using unimods or user-defined modifications"
- "it can only have the minimal amound of items" --> it can have the minimal amount of items (i.e.: ...)
- "vallues" --> values